packages/gasket-preset-nextjs/lib/preset-prompt.js

```diff
@@ -1,3 +1,96 @@
+function prompts(context, prompt) {
```

Comment:
Why were the Nextjs prompts moved to the preset-prompt? Would this lead to code duplication in the webapp preset and any other preset in the future that uses the next plugin.

Reply:
They were moved because there was duplication. They should be removed from the plugin itself.

Reply:
@jpina1-godaddy not sure if you saw this!

Reply:
Where was the duplication? Weren't these only called in the nextjs plugin? 

If we want to use some of these prompts for the webapp preset, for example the promptAppRouter prompt, wouldn't we have to copy and paste this promptAppRouter prompt code from here into the webapp preset instead of sharing the code between the two presets if it was located in the nextjs plugin?

Reply:
Yea this change doesn't seem correct.

The general rule should be:
- Preset prompts present choices of which plugins to use.
- Plugin prompts present choices of which features to use.

If my present includes the a plugin, I expect that plugin to ask about what features to set up.
If I want to give my users options between different plugins, then I add that choice as a prompt to my preset. After that, if there are any feature choices, the plugin should ask about it.

So, unless I'm missing or overlooking something here, these Next.js prompts should remain in the Next.js plugin.

Reply:
Oh, it might be an ordering issue?

1. Preset prompts fire
2. Next plugin asks if you want Custom Server
3. if you do, then Preset can no longer as if you want Express or Fastify

Is that the scenario here?

Reply:
After a team chat, this is going to be explored further to define a good pattern moving forward. Approving for now to unblock.